src/components/styles/Join.scss

```diff
@@ -0,0 +1,56 @@
+@import 'variables';
+
+// same as team-page, refactor?
+.join-page {
+  margin-top: calc(#{$navbar-height});
+  background-color: #fff;
+  padding: 24px $padding-horiz;
+}
+
```

Comment:
LGTM, just a small nit: on larger screens the margins of the content on the frontpage are padded so that it doesn't start from too far to the left, so it's a little jarring when the user goes to the Join page:

|Frontpage|Join|
|---|---|
|![Screenshot_20201128_005526](https://user-images.githubusercontent.com/30734384/100494917-78291f80-3114-11eb-935e-e17fffe2cf5f.png)|![Screenshot_20201128_005539](https://user-images.githubusercontent.com/30734384/100494916-765f5c00-3114-11eb-960a-805a8e3f11fa.png)|

(It's not a problem for the Team page since the nature of that page's content is different.)

Easy fix though, just add this:
```suggestion

.join-page > * {
 max-width: $max-section-width;
 margin: 0 auto;
}

```

Reply:
thanks arjun 🙏